lib/diagnostic.js

```diff
@@ -8,38 +8,57 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-const min = function min() {
-
+const min = function min(array, init) {
+  let result;
+  if (init !== undefined){
+    result = init;
+  }
+  return result;
 };
 
 // Question 2
 // Write a callback for reduce that produces
 // the sum of the values in the array
-const sum = function sum() {
 
+const sum = function(prev, curr){
+  return prev + curr;
 };
 
 // Question 3
 // Write a function that returns true
 // if invoking the callback for any element
 // in the array returns a truthy value
 const any = function any(array, callback) {
-
+  for (let i = 0; i < array.length; i++) {
+    if(array[i]){
+      return true;
+    }
+  }
 };
```

Comment:
So close here. You'd just need to return `false` from the function in the event nothing is true.